Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -129,6 +129,10 @@ public void run() {
             mTagsBucket = mSimperium.bucket(tagSchema);
             mPreferencesBucket = mSimperium.bucket(new Preferences.Schema());
             mAccountBucket = mSimperium.bucket(new Account.Schema());
+            // Setup Account Verification Watcher to listen to network changes on the account bucket
+            VerificationListener verificationListener = new VerificationListener();
+            mAccountVerificationWatcher = new AccountVerificationWatcher(this, verificationListener);
+            mAccountBucket.addOnNetworkChangeListener(mAccountVerificationWatcher);
```

Comment:
given that we don't use any of these apart from starting the whole chain, would you consider not storing them as variables?

```java
mAccountBucket.addOnNetworkChangeListener(new AccountVerificationWatcher(this, new VerificationListener()));
```

Reply:
Fixed in f7793d3831cc70952383bbf2c9307d41394b1839.